crates/sui-prover/src/prove.rs

```diff
@@ -41,7 +41,7 @@ impl From<BuildConfig> for MoveBuildConfig {
             additional_named_addresses: config.additional_named_addresses,
             save_disassembly: false,
             implicit_dependencies: BTreeMap::new(),
-            modes: vec![ModeAttribute::VERIFY_ONLY.into(), ModeAttribute::TEST_ONLY.into(), ModeAttribute::TEST.into()],
```

Comment:
can we keep `ModeAttribute::TEST_ONLY` and remove `ModeAttribute::TEST`? Would that result in including `#[test_only]` functions without any of the test poison stuff?